### ISSUE TITLE
Missing dev.forkhandles:parser4k dependency in http4k-connect-amazon-dynamodb-fake:5.26.0.0

### ISSUE THREAD

**Issue by obecker** [http4k/http4k-connect#484]
After upgrading to 5.26.0.0 I see `ClassNotFoundException`s for `parser4k.OutputCache` in my tests.

I guess this comes from changing the parser4k dependency in http4k-connect-amazon-dynamodb-fake from
```xml
<dependency>
  <groupId>dev.forkhandles</groupId>
  <artifactId>parser4k</artifactId>
  <scope>runtime</scope>
</dependency>
```

to
```xml
<dependency>
  <groupId>dev.forkhandles</groupId>
  <artifactId>parser4k</artifactId>
  <scope>provided</scope>
</dependency>
```

Is this intentional?

**Comment by daviddenton** [comment]
nope. That's a bug. We've refactored the build this week and it's crept in. Thanks for raising it.

**Comment by daviddenton** [comment]
This should be fixed in 5.26.0.2 :)
